workflow/rules/common.smk

```diff
@@ -38,7 +38,7 @@ validate(units, schema="../schemas/units.schema.yaml")
 def get_cutadapt_input(wildcards):
     unit = units.loc[wildcards.sample].loc[wildcards.unit]
 
-    if pd.isna(unit["fq1"]):
+    if pd.isna(unit["fq1"]).all():
```

Comment:
`unit["fq1"]` should really be a single cell from the units table, so no summary function should be necessary. I would rather go for calling [`squeeze()`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.squeeze.html) to ensure that we get a scalar value out:
```suggestion
    if pd.isna(unit["fq1"].squeeze()):
```

I think this applies to all instances of using `.all()`, below, as well.